Client.xcodeproj/project.pbxproj

```diff
@@ -18107,6 +18127,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = 43AQ936H96;
+				EXCLUDED_SOURCE_FILE_NAMES = "Client/Nimbus/TestData/*";
```

Comment:
The `TestData` is living under the Client target, ideally that's not there. Does what they are currently discussing in the iOS eng meeting about database could apply here?

Reply:
It is not possible to move them anyplace else as this is an experiment that needs to be applied during testing. Because of the way UITests are running the data has to be in Client. To not have this in the prod build I excluded the folder with the file from anything but the Fennec scheme.